app/models/application_model.rb

```diff
@@ -0,0 +1,18 @@
+class ApplicationModel
```

Comment:
I don't believe we need this class anymore.

Reply:
true